source/ore/publish.rst

```diff
@@ -0,0 +1,139 @@
+To do this, you can navigate to your profile and click the key symbol next to your avatar.
+
+.. image:: /images/ore/help_1.png
+    :align: center
+    :alt: PGP public key 1
+
+You will then be prompted to enter your key into a text box. You must be sure to enter the key in it's entirety or
+signature validation will fail.
+
+.. image:: /images/ore/help_2.png
+    :align: center
+    :alt: PGP public key 2
+
+Finally you must sign your built plugin with your *private* key of the public key you uploaded. You only need to sign
+the file that you are uploading: for instance, if you are uploading a ZIP with your plugin JAR included, only the ZIP
+needs to (and should only be) signed. The easiest way to sign the file you intend to upload is with the following GPG
+command.
+
+.. code-block:: bash
+
+    gpg --output MyPlugin-signed.jar --sign MyPlugin.jar
+
+.. note::
+
+    If you're using build manager such as Gradle or Maven, there is are plugins you can apply to your build script that
+    can sign built artifacts and automate this laborious process for you. For more information, please refer to
+    `Gradle's documentation <https://docs.gradle.org/current/userguide/signing_plugin.html>`__ or
+    `Maven's documentation <http://maven.apache.org/plugins/maven-gpg-plugin/>`__.
+
+Uploading Your Plugin
+~~~~~~~~~~~~~~~~~~~~~
+
+Once your plugin's JAR file is packaged with an ``mcmod.info`` descriptor file in the top-level and signed with your
+PGP private key, your plugin is ready for uploading! To create a project on Ore, you must have an active Sponge
+account. Hitting the "Sign up" button in the top-right corner will take you to the appropriate page to create one. If
+you already have an account, just hit the "Log in" button in the top-right corner to log into Ore.
+
+Once logged in, navigate to your avatar and select the "New" option in the drop-down menu that appears, or just press
+the "C" key.
+
+Setting Up Your Project on Ore
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After uploading your plugin, you will be presented with some settings to configure your project however you like. Don't
+worry, all of these can be changed later in the settings panel. After setting up your project you will be able to invite
+Sponge users to be a part of your project. Any user that is registered on the forums can be invited to a project even if
+they have never logged onto Ore. Upon creation of your project, the users you have invited will receive a notification
+and will be able to either accept or decline your invitation. Project members will not be visible to the public until
+the invitation has been accepted.
+
+There are currently three groups you may assign to project members: `Developer`, `Editor`, and `Support`. The user
+who created the project will be assigned the `Owner` which has unrestricted access to the project. Here is a quick
+rundown of what each group can do within your project.
```

Comment:
We should probably add a explanation for the `Owner` role as well.
* Is it transferable?
* Can multiple people have it?
* Is he the only role that can assign member roles?